kv/txn_coord_sender_test.go

```diff
@@ -270,14 +217,13 @@ func TestTxnCoordSenderMultipleTxns(t *testing.T) {
 	defer s.Stop()
 	defer teardownHeartbeats(s.Sender)
```

Comment:
Could you do that in a separate change? I'm feeling a bit out of touch with even this PR due to the down time between my windows of opportunity to work on things.